x-pack/plugins/fleet/common/constants/plugin.ts

```diff
@@ -6,3 +6,4 @@
  */
 
 export const PLUGIN_ID = 'fleet';
+export const INTEGRATIONS_PLUGIN_ID = 'integrations';
```

Comment:
Do we plan to move integrations to a different plugin? or it will always be an app inside the fleet plugin?

Reply:
I think the plan would be to eventually move it to a separate plugin entirely.

Reply:
later on it may become separate, but that's not firmly decided at this point